lib/vtt.js

```diff
@@ -1204,8 +1204,17 @@
     })();
   };
 
-  WebVTT.Parser = function(window, decoder) {
+  WebVTT.Parser = function(window, vttjs, decoder) {
```

Comment:
I'm ok with making that change. Was just keeping it like this so the API won't change, especially since it's optional.
Yeah, shims might be ok.